index.js

```diff
@@ -64,7 +64,7 @@ class Aperture {
 
       const recorderOpts = {
         destination: fileUrl(this.tmpPath),
-        fps,
+        framesPerSecond: fps,
```

Comment:
Why not keep this variable as `fps` as it's named in the user facing apis? I think that would make it easier to keep track of 👍

Reply:
I intend to eventually change the public API to `framesPerSecond`, but I just did a major release, so not worth doing that yet. That's why it's using `framesPerSecond` internally as much as possible.

Reply:
Why not keep it as `fps` for simplicity?

Reply:
Clarity > Simplicity